README.md

```diff
@@ -143,6 +143,28 @@ $ clang \
 The FPU selection can be skipped, but it is not recommended to as the defaults
 are different to GCC ones.
 
+
+The builds of the toolchain come packaged with two config files, Omax.cfg and OmaxLTO.cfg.
+When used, these config files enable several build optimisation flags to achieve highest performance on typical embedded benchmarks. OmaxLTO.cfg enables link-time optimisation (LTO) specific flags.
+These configs can be optionally passed using the `--config` flag. For example:
+
+```
+$ clang \
+example.c \
+...
+--config=Omax.cfg \
+--config=OmaxLTO.cfg \
+-o example
+```
+
+Users should be warned that Omax.cfg enables `-ffast-math` which breaks IEEE compliance and
+enables maths optimisations which can affect code correctness.  LTOs are
+kept separately in OmaxLTO.cfg as users may not want LTOs due to potential increase in link time
+and/or increased memory usage during linking. The flags used in these configs are used internally by
+us at Arm, information on what they do can be found at
```

Comment:
Thank you for the update! Sorry, I was not clear: I meant that these are clang internal options (-mllvm ones) and I believe they are not documented - best I was able to find for some of them was comments in the source code. Maybe this is the way to put it: config files use llvm internal undocumented options, please see source code of corresponding optimization passes in LLVM project to find out more.

Reply:
Ah, I see. Updated